newsapi_headlines-to-email/main.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/log;
```

Comment:
Code is not formatted properly.

Reply:
Updated in [5131705](https://github.com/ballerina-guides/integration-samples/pull/35/commits/51317055927a041c645786b20bcb6af3864f1081)